scripts/changelog.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ $1 ]; then
+
+  if [ $2 ]; then
+    HEAD=$2
+  else
+    HEAD=head
+  fi
+  git log '--format=format:* [``%h``](https://github.com/slick/slick/commit/%H) %s' --no-merges $1..$2
```

Comment:
You do not actually use the variable `HEAD` did you mean to end the line with `$1..$HEAD`

Reply:
Same holds in the other script